src/app/forms/components/test-type-select/test-type-select.component.html

```diff
@@ -0,0 +1,53 @@
+<div class="govuk-summary-list">
+  <ng-container [formGroup]="form">
+    <ng-container formArrayName="testTypes">
+      <ng-container formGroupName="0">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">Test name:</dt>
+          <dd id="test-testTypeId" class="govuk-summary-list__value">
+            {{ testTypeIdControl?.value | testTypeName: (selectAllTestTypes$ | async) }}
+            <app-button *ngIf="isEditing" class="float-right" type="underline" (clicked)="handleChange()">{{
+              edit ? 'Cancel' : 'Change'
+            }}</app-button>
+          </dd>
+        </div>
+      </ng-container>
+    </ng-container>
+  </ng-container>
+</div>
+
+<div *ngIf="edit" class="categories">
+  <div class="list_container" [ngClass]="{ 'list_container--shrink': categories.length > 0 + 1 }">
+    <ol class="list">
+      <li
+        *ngFor="let type of selectAllTestTypes$ | async; trackBy: tackByFn"
+        class="list__item"
+        [ngClass]="{ 'list__item--active': isSelected(type.id!!) }"
+      >
+        <button (click)="handleCategory(type, 0)">
+          {{ type.id }}: {{ type.name }} <span [ngClass]="{ 'list__item--more': hasNext(type) }"></span>
```

Comment:
Do we need to show the TestTypeId (`type.id`) to the user?  (and again on line 47)

```suggestion
          {{ type.name }} <span [ngClass]="{ 'list__item--more': hasNext(type) }"></span>
```